src/ios-deploy/ios-deploy.m

```diff
@@ -1198,18 +1208,30 @@ void read_dir(AFCConnectionRef afc_conn_p, const char* dir,
         return;
     }
 
+    // Show the file size after the file name in verbose mode
+    long file_size = 0;
     while((AFCKeyValueRead(afc_dict_p,&key,&val) == 0) && key && val) {
         if (strcmp(key,"st_ifmt")==0) {
             not_dir = strcmp(val,"S_IFDIR");
-            break;
```

Comment:
We probably still want to break when we've found the values we need (I'm not sure how long this list is normally).